README.md

```diff
@@ -37,12 +37,12 @@ await inference.textToImage({
 
 This is a collection of JS libraries to interact with the Hugging Face API, with TS types included.
 
-- [@huggingface/inference](packages/inference/README.md): Use the Inference API to make calls to 100,000+ Machine Learning models, or your own [inference endpoints](https://hf.co/docs/inference-endpoints/)!
+- [@huggingface/inference](packages/inference/README.md): Use Inference Endpoints (serverless) to make calls to 100,000+ Machine Learning models
 - [@huggingface/hub](packages/hub/README.md): Interact with huggingface.co to create or delete repos and commit / download files
 - [@huggingface/agents](packages/agents/README.md): Interact with HF models through a natural language interface
 
 
-With more to come, like `@huggingface/endpoints` to manage your HF Endpoints!
+With more to come, like `@huggingface/endpoints` to manage your dedicated Inference Endpoints!
 
```

Comment:
```suggestion
```
From the updated readme, the package is supposed to work with both serverless and dedicated flavors, so I would remove this, no?